yash-datalake/nested/streamAnalytics.json

```diff
@@ -0,0 +1,462 @@
+      "name": "[parameters('DataAggregatorStreamAnalyticsJob1')]",
+      "apiVersion": "2019-06-01",
+      "location": "[parameters('location')]",
+      "properties": {
+        "sku": {
+          "name": "standard"
+        },
+        "outputErrorPolicy": "stop",
+        "eventsOutOfOrderPolicy": "adjust",
+        "eventsOutOfOrderMaxDelayInSeconds": 0,
+        "eventsLateArrivalMaxDelayInSeconds": 5,
+        "dataLocale": "en-US",
+        "inputs": [
+          {
+            "name": "EventHub",
+            "properties": {
+              "type": "stream",
+              "serialization": {
+                "type": "JSON",
+                "properties": {
+                  "encoding": "UTF8"
+                }
+              },
+              "datasource": {
+                "type": "Microsoft.ServiceBus/EventHub",
+                "properties": {
+                  "serviceBusNamespace": "[parameters('eventHubNamespaceName')]",
+                  "sharedAccessPolicyName": "RootManageSharedAccessKey",
+                  "sharedAccessPolicyKey": "[listkeys(resourceId('Microsoft.EventHub/namespaces/authorizationRules', parameters('eventHubNamespaceName'), variables('defaultSASKeyName')), '2017-04-01').primaryKey]",
+                  "eventHubName": "[parameters('DataCleaningOutputStream')]"
+                }
+              }
+            }
+          }
+        ],
+        "transformation": {
+          "name": "Transformation",
+          "properties": {
+            "streamingUnits": "[variables('numberOfStreamingUnits')]",
+            "query": "SELECT \n System.Timestamp AS allEvents, \n COUNT(sku) AS allsku \n INTO \n Blob \n FROM \n EventHub \n GROUP BY \n TumblingWindow(second,10) \n \n SELECT \n System.Timestamp AS allEvents, \n COUNT(sku) AS allsku \n INTO \n PBI \n FROM \n EventHub \n GROUP BY \n TumblingWindow(second,10)"
+          }
+        },
+        "outputs": [
+          {
+            "name": "PBI",
+            "properties": {
+              "dataSource": {
+                "type": "PowerBI",
+                "outputPowerBISource": {
+                  "dataSet": "Quickstart-sellingProductPerMin-Dataset",
+                  "table": "sellingProductPerMin-Table",
+                  "groupName": "My Workspace",
+                  "refreshToken": "dummytoken",
+                  "tokenUserDisplayName": "anurag.doe@git.com",
+                  "tokenUserPrincipalName": "anurag.doe@git.com"
+                },
+                "properties": {
+                  "dataSet": "Quickstart-sellingProductPerMin-Dataset",
+                  "table": "sellingProductPerMin-Table",
+                  "groupName": "My Workspace",
+                  "refreshToken": "dummytoken",
+                  "tokenUserDisplayName": "anurag.doe@git.com",
+                  "tokenUserPrincipalName": "anurag.doe@git.com"
+                }
+              },
+              "serialization": null,

```

Comment:
remove all empty/null properties from all templates

Reply:
Done with removing all empty/null properties from all the templates.